airflow/contrib/operators/mysql_to_gcs.py

```diff
@@ -208,12 +211,13 @@ def convert_types(cls, value):
             return value
 
     @classmethod
-    def type_map(cls, mysql_type):
+    def type_map(cls, mysql_type, mysql_type_flag):
         """
         Helper function that maps from MySQL fields to BigQuery fields. Used
         when a schema_filename is set.
         """
         d = {
+            FIELD_TYPE.INT24: 'INTEGER',
```

Comment:
nit: the `TIME` type defaults to STRING before this PR, but it is considered a binary field, so this change will make it default to 'BYTES'.